app/mailers/distribution_mailer.rb

```diff
@@ -27,7 +27,10 @@ def partner_mailer(current_organization, distribution, subject, distribution_cha
     pdf = DistributionPdf.new(current_organization, @distribution).compute_and_render
     attachments[format("%s %s.pdf", @partner.name, @distribution.created_at.strftime("%Y-%m-%d"))] = pdf
     cc = [@partner.email]
-    cc.push(@partner.profile&.pick_up_email) if distribution.pick_up?
+    if distribution.pick_up? && @partner.profile&.pick_up_email
+      pick_up_emails = @partner.profile.pick_up_email.delete(" ").split(",")
+      (cc << pick_up_emails).flatten! if pick_up_emails
```

Comment:
This is trying to be a bit too concise IMO. I'd probably move `flatten!` to line 34 before `compact!`.